slides/control_structures.tex

```diff
@@ -0,0 +1,183 @@
+	\end{figure}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Matrix of the Control Structures}
+	\begin{figure}
+		\input{../assets/figures/cube_yap.tex}
+	\end{figure}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Competitive Creation}
+	\begin{columns}
+		\begin{column}{0.5\textwidth}
+			\begin{figure}
+				\input{../assets/figures/competitive_creation_incr.tex}
+				\caption*{(a) Increasing marginal costs}
+			\end{figure}
+		\end{column}
+		\begin{column}{0.5\textwidth}
+			\begin{figure}
+				\uncover<2->{
+					\input{../assets/figures/competitive_creation_const.tex}
+					\caption*{(b) Constant marginal costs}
+				}
+			\end{figure}
+		\end{column}
+	\end{columns}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Monopolistic Creation}
+	\begin{figure}
+		\input{../assets/figures/monopolistic_creation.tex}
+		\caption*{Monopolistic money creation with constant marginal costs}
+	\end{figure}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Physical Representation}
+	Physical monetary units are tied to an object.
+	\vspace{1em}
+	\begin{figure}
+		\centering
+		\includegraphics[height = 0.15\textheight]{../assets/images/currency}
+	\end{figure}
+	\vspace{1em}
+	\begin{columns}
+		\begin{column}{0.5\textwidth}
+			\uncover<2->{
+				\textbf{Pros:}
+				\begin{itemize}
+					\item Simple
+					\item Clear ownership rights
+					\item Anonymity
+					\item No systemic dependency
+				\end{itemize}
+			}
+		\end{column}
+		\begin{column}{0.5\textwidth}
+			\uncover<3->{
+				\textbf{Cons:}
+				\begin{itemize}
+					\item Location bound
+					\item Safekeeping and transport
+					\item Counterfeiting
+					\item Divisibility
+				\end{itemize}
+			}
+		\end{column}	
+	\end{columns}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Virtual Representation}
+	Virtual monetary units allow the transfer of value without any change in the physical control of a particular object.
+	\vspace{1em}
+	\begin{figure}
+		\centering
+		\includegraphics[height = 0.15\textheight]{../assets/images/virtual}
+	\end{figure}
+	\vspace{1em}
+	\begin{columns}
+		\begin{column}{0.5\textwidth}
+			\uncover<2->{
+				\textbf{Pros:}
+				\begin{itemize}
+					\item No physical handover
+				\end{itemize}
+			}
+		\end{column}
+		\begin{column}{0.5\textwidth}
+			\uncover<3->{
+				\textbf{Cons:}
+				\begin{itemize}
+					\item Contestable
+				\end{itemize}
+			}
+		\end{column}	
+	\end{columns}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Transaction Processing}
+	\begin{columns}
+		\begin{column}{0.5\textwidth}
+			\textbf{Decentralized}
+			\begin{figure}
+				\begin{flushleft}
+					\input{../assets/figures/dec_process.tex}
+				\end{flushleft}
+			\end{figure}
+		\end{column}
+		\begin{column}{0.5\textwidth}
+			\uncover<2->{
+				\textbf{Centralized}
+				\begin{figure}
+					\begin{flushleft}
+						\input{../assets/figures/cen_process.tex}
+					\end{flushleft}
+				\end{figure}
+			}
+		\end{column}
+	\end{columns}
+	\vspace{1.5em}
+	\uncover<3->{
+	For virtual monetary units:
+	\begin{center}
```

Comment:
Not sure, if we're duplicating content now with this table to "payment systems*
From a format perspective I'd recommend
- Adding more vspace
- Title in bold 
- question mark in table in focus color

Reply:
This is definitely something we should double check before recording starts, but in my opinion duplicating the content is not a problem here. One video talks in detail about payment systems whereas the other video simply recaps the content to cohesively categorize Bitcoin along the three monetary control structures.

Reply:
I added more vspace and used focus color for the question mark, but I left the text above the table as I view it as a transition and not as a title. Captions should be used below the tables and figures.